frontend/task/TaskAbout.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {getTaskMetadata} from "./platform/platform";
+import {getMessage} from "../lang";
+import {useAppSelector} from "../hooks";
+import {AppStore} from "../store";
+
+export const selectDisplayAbout = (state: AppStore) => {
+    return state.task.currentTask || (state.player && state.player.data && state.player.data.version && Number(state.player.data.version.split('.')[0]) < 7);
+}
+
+export function TaskAbout() {
+    const metadata = getTaskMetadata();
+
+
+    const recordingVersion = useAppSelector(state => {
+        if (state.player && state.player.data && state.player.data.version) {
+            let versionComponents = state.player.data.version.split('.').map(Number);
+
+            return versionComponents[0];
+        }
+
+        return null;
+    });
+
+    if (7 > recordingVersion) {
+        return (
+            <div className="task-license">
```

Comment:
This is already a special case for backwards compatibility since the new recordings now have a version >= 7. This code should therefore not change in the future.